### ISSUE TITLE
Learning rate when epochs are small

### ISSUE THREAD

**Issue by waner1998** [CUHK-AIM-Group/U-KAN#36]
Dear Author,

Thank you for your impressive work. In your paper, the number of epochs is set to 400. However, when I tried to reproduce your code with a smaller number of epochs (e.g., 50), I found it difficult for the model to converge. I was only able to achieve convergence by increasing the learning rate to 1e-2.

May I ask what learning rate you would recommend for training SEG_UKAN when using a smaller number of epochs?

Best regards,
Liu.
